src/page/CartPage/index.jsx

```diff
@@ -0,0 +1,116 @@
+// @ts-nocheck
+import { useCallback, useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import { Image, CartProductItem, CheckBox, TotalPrice } from 'components';
+
+import store from 'store/store';
+import {
+  doAddProductToOrder,
+  doInitializeOrder,
+  doSelectiveDeleteFromCart,
+} from 'actions/actionCreator';
+import empty from 'assets/empty.jpeg';
+import Styled from 'page/CartPage/index.style';
+import { useNavigate } from 'react-router-dom';
+import useSnackbar from 'hooks/useSnackbar';
+import { MESSAGE } from 'utils/constants';
+
+const CartPage = () => {
+  const [renderSnackbar] = useSnackbar();
+  const navigate = useNavigate();
+  const { isAuthenticated } = useSelector(state => state.authReducer);
```

Comment:
이미 merge가 돼서, 새로고침 시 로그인 유지 불가 에러에 대해 수정한 코드를 코멘트로 적어두겠습니다.
```suggestion
  const { isLoading, isAuthenticated } = useSelector(state => state.authReducer);
```